graph-manager-docs/source/schema-validation.mdx

```diff
@@ -1,36 +1,200 @@
 ---
-title: Validating schema changes
-description: Check if schema changes are safe or breaking by comparing against live server traffic
+title: Schema validation
+description: Ensure the safety of changes to your production graph
 ---
 
-There are many types of schema changes that can be potentially breaking to clients, like removing a field, if made without special consideration. For safety, some organizations take the approach of _never_ making these types of changes, but this leads to an ever-growing schema and reduced API flexibility over time. In reality, making these types of changes to a schema can be very safe as long as you have tools in place to ensure that no queries are broken in the process.
+Certain changes to your GraphQL schema (such as removing a field or type) can potentially break one of your application's clients. Apollo Graph Manager provides **schema validation** to help you identify breaking changes before you make them, and to help you identify when a change _won't_ break anything.
```

Comment:
Do we seriously never say that this is a paid feature in this doc?

Reply:
We say it's part of Team and Enterprise plans, but that's clearly insufficient. Added a bolded indicator to that callout.